docs/Developers/Writing_Good_Commits.rst

```diff
@@ -71,7 +71,7 @@ Example:
 
 ::
 
-    setup: Install .coafile via package_data
+    setup.py: Change bears' entrypoint
```

Comment:
I would suggest you make these changes in another commit. Describing why you are making these changes. Make this commit about only a single thing please.